src/doc/rustc/src/platform-support/openharmony.md

```diff
@@ -0,0 +1,133 @@
+# `target-name-here`
```

Comment:
Please don't forget to also update https://github.com/rust-lang/rust/blob/HEAD/src/doc/rustc/src/platform-support.md and add this file to https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/SUMMARY.md.

Reply:
Done.